pkg/manager/manager.go

```diff
@@ -1,18 +1,19 @@
-package main
+package manager
 
 import (
 	"bytes"
 	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
+	utils "github.com/openshift/ci-chat-bot/pkg"
```

Comment:
Should be named `util` or be unnamed for consistency with the other files in the project.